can/parser_pyx.pyx

```diff
@@ -184,9 +184,7 @@ cdef class CANDefine():
       defs = def_val[1::2]
 
       if address not in dv:
-        dv[address] = {}
         msgname = address_to_msg_name[address]
```

Comment:
By the way, this looks like a bug.    The code below references `msgname` even when this `if` block is not taken, so it will get the previous iteration's value.  (If that's really the intention, I'll add a comment about it.)

Reply:
yeah, looks like a bug